instrumentation/opentelemetry-instrumentation-sqlalchemy/tests/test_sqlalchemy.py

```diff
@@ -248,6 +248,7 @@ def test_uninstrument(self):
 
         self.memory_exporter.clear()
         SQLAlchemyInstrumentor().uninstrument()
```

Comment:
Please dont call `uninstrument` on a new SQLAlchemyInstrumentor instance,
suggested code:
```js
def test_uninstrument(self):
    engine = create_engine("sqlite:///:memory:")
    instrumentor = SQLAlchemyInstrumentor()
    instrumentor.instrument(
        engine=engine,
        tracer_provider=self.tracer_provider,
    )
    cnx = engine.connect()
    cnx.execute("SELECT	1 + 1;").fetchall()
    spans = self.memory_exporter.get_finished_spans()

    self.assertEqual(len(spans), 2)
    # first span - the connection to the db
    self.assertEqual(spans[0].name, "connect")
    self.assertEqual(spans[0].kind, trace.SpanKind.CLIENT)
    # second span - the query itself
    self.assertEqual(spans[1].name, "SELECT :memory:")
    self.assertEqual(spans[1].kind, trace.SpanKind.CLIENT)

    self.memory_exporter.clear()
    instrumentor.uninstrument()
    cnx.execute("SELECT	2 + 2;").fetchall()
    spans = self.memory_exporter.get_finished_spans()
    self.assertEqual(len(spans), 0)
```

Reply:
I'm not sure, can you explain what's the difference?

Reply:
We dont want to create additional instrumentor class with new connection and new engine,
we would like to see that we remove listener events for specific instrumentation with specific engine

Reply:
It will not create another sqlalchemy instrumentor, if I understand this correctly it's like Singelton:
https://github.com/open-telemetry/opentelemetry-python-contrib/blob/c92ba14316a90db2a8e8a7702aa21ce114a581a5/opentelemetry-instrumentation/src/opentelemetry/instrumentation/instrumentor.py#L50
What do you think?